ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -264,6 +266,47 @@ func (s *Service) Up() error {
 		return servicediscoveryUpdate(aws.StringValue(newTaskDefinition.NetworkMode), entity.GetServiceName(s), s.Context())
 	}
 
+	// add new tags, if provided
+	if tagVal := s.Context().CLIContext.String(flags.ResourceTagsFlag); tagVal != "" {
+		err = s.updateTags(ecsService, tagVal)
+		if err != nil {
+			return err
+		}
+		log.Info("Added new tags to service")
+	}
+
+	return nil
+}
+
+// Allows us to inject the mock tagging client in tests
+// This isn't the best design practice, but its convenient since we only need
```

Comment:
you can probably omit this and the follow line of commentary :P (but leave the TODO)